src/modules/addComment.js

```diff
@@ -0,0 +1,33 @@
+import getcomments from './getComments.js';
+import displayComments from './displayComments';
+
+const invApiComment = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/6jTH3jOos7OpnHxCg6lz/comments/';
+
+export default (button, username, text, bookId) => {
+  button.addEventListener('click', async (e) => {
+    e.preventDefault();
+    bookId = bookId.toString();
+    const uname = username.value.trim();
+    const ucomment = text.value.trim();
```

Comment:
It is advisable to use Camel case naming convention while declaring variables as it helps with readability